openphish/help.md

```diff
@@ -5,14 +5,11 @@ It identifies phishing sites and performs intelligence analysis in real time wit
 
```

Comment:
Using the OpenPhish plugin for Rapid7 InsightConnect within an automation workflow can help aid in phishing investigations with minimal effort.